src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -84,21 +84,24 @@ export class SchoolSelectField extends React.Component {
 
   handleOptionClick = ({ address1, address2, address3, city, facilityCode, name, state, zip, country }) => {
```

Comment:
Alphabetize this argument list.

Reply:
Ha, should I admit I was looking at the order of this list and thought it wasn't a logical way to order address info? Alphabetical didn't even enter my mind.